.github/workflows/deploy_environment.yml

```diff
@@ -10,12 +10,17 @@ on:
       commit-identifier:
         description: SHA or tag to apply, (optional) when omitted, the head of the selected branch is used
         type: string
-        required: true
+        required: false
       release-tag:
         description: release name to apply to resources
         type: string
         required: false
         default: unnamed
+      oidc-domain:
+        description: 'OIDC Domain. Defaults to "core" when omitted'
+        required: false
+        type: string
+        default: 'core'
```

Comment:
Great update. Please log a backlog story to remove this default. I know we don't want to break things right now. In previous conversations Brian felt we should not be defaulting values like this in the input.

Reply:
https://opensesame.atlassian.net/browse/CORE-4196